### PR TITLE
USEID-589/tidy up documentation

### PR DESCRIPTION
**Pull request by mpanne**
Cleanup documentation:

- Remove sequence diagram for desktop flows
- Add sequence diagram for Magnus mobile only flow
- Update Magnus integration guide to fit for mobile only flow

### CODE REVIEW

**Review by code28 — CHANGES_REQUESTED** [review]
Just tiny comments, otherwise: 👍

**Review by hannesschaletzky — APPROVED** [review]

File: doc/eID/eid-integration-magnus.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # UseID Integration
```
[the remark this answers is not in the document]
Comment (mpanne) [reply doc/eID/eid-integration-magnus.md:1]: For now we will leave it as is and use "UseID". See [discussion in Slack](https://digitalservicebund.slack.com/archives/C0314PBC8U8/p1666086966535679).

File: doc/eID/eid-integration-magnus.md [hunk 2]
```diff
@@ -20,18 +20,12 @@ and including this script in the page:
 
 As value for `data-tc-token-url` the already known tcTokenURL is used.
 
-> The eService usually presents a link to the user which points to the eID client application (e.g. AusweisApp2) including the tcTokenURL as a query parameter. With UseID, this is replaced by the web widget displaying a QR code which can be scanned to start the UseID mobile app.
+> The eService usually presents a link to the user which points to the eID client application (e.g. AusweisApp2, BundesIdent) including the tcTokenURL as a query parameter. With UseID, this is handled by the web widget.
```
[the remark this answers is not in the document]
Comment (mpanne) [reply doc/eID/eid-integration-magnus.md]: Added something.

File: doc/eID/eid-integration-magnus.md [hunk 3]
```diff
@@ -20,18 +20,12 @@ and including this script in the page:
...
 
 ### 2. ID process
 
-The underlying eID technology has not changed for the eService, including the process steps of scanning the identification document and PIN entry. Changes for the users are in using the UseID mobile app on their smartphones as scanning devices.
+The underlying eID technology has not changed for the eService, including the process steps of scanning the identification document and PIN entry. Changes for the users are in using the BundesIdent mobile app on their smartphones as scanning devices.
 
 ### 3. Success and Refresh
 
 The user is returned to the `refreshAddress` of the eService, which is provided in the TC token, on the device displaying the widget.
```
[the remark this answers is not in the document]
Comment (mpanne) [reply doc/eID/eid-integration-magnus.md]: Good catch.

**Review by code28 — APPROVED** [review]

**Pull request merged**